create_static_html_files.py

```diff
@@ -305,80 +307,81 @@ def generate_plugin_html(row, template, plugin_dir):
     # Save the HTML file for each plugin
     os.makedirs(plugin_dir, exist_ok= True)
     file_name = f"{row['name']}.html"
-    with open(plugin_dir + file_name, 'w') as file:
+    with open(f'{plugin_dir}/{file_name}', 'w') as file:
         file.write(filled_template)
 
 
 
-#########################################
-## main starts
-#########################################
+if __name__ == "__main__":
+    build_dir = sys.argv[1] if len(sys.argv) > 1 else '.'
+    data_dir = f"{build_dir}/data"
+    static_dir = f"{build_dir}/static"
+    plugin_dir = f"{build_dir}/plugins"
+    template_dir = f"{build_dir}/templates"
```

Comment:
nit: I would recommend single quotes

Perhaps add precommit and run through ruff.